pkgs/applications/emulators/wine/sources.nix

```diff
@@ -65,6 +65,16 @@ let
     source ${./update-lib.sh}
   '';
 
+  # Needed for wine versions < 10.2 to fix compatibility with binutils 2.44
+  # https://github.com/NixOS/nixpkgs/issues/399714
+  # https://bugs.winehq.org/show_bug.cgi?id=57819
+  # https://gitlab.winehq.org/wine/wine/-/merge_requests/7328
+  patch-binutils-2_44-fix-wine-older-than-10_2 = pkgs.fetchpatch {
+    name = "binutils-2.44-fix-wine-older-than-10.2";
+    url = "https://gitlab.winehq.org/wine/wine/-/merge_requests/7328.patch";
```

Comment:
Usually we don't allow referencing PRs directly for patches, since they could change. But this PR is merged so it might be okay.

Reply:
In which case we should fetch the commit by id.

Reply:
Since it's (only) two commits I referenced the commits directly, if that's acceptable?